### PR TITLE
Added DockerBuildManagement and SwarmManagement

### PR DESCRIPTION
**Pull request by hansehe**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -533,6 +534,7 @@ Services to securely store your Docker images.
 - [forward2docker](https://github.com/bsideup/forward2docker) :skull: - Utility to auto forward a port from localhost into ports on Docker containers running in a boot2docker VM by [@bsideup](https://github.com/bsideup)
 - [Lando](https://github.com/lando/lando) - Lando is for developers who want to quickly specify and painlessly spin up the services and tools needed to develop their projects. By [Tandem](https://thinktandem.io/)
 - [Vagga](https://github.com/tailhook/vagga) - Vagga is a containerisation tool without daemons. It is a fully-userspace container engine inspired by Vagrant and Docker, specialized for development environments by [@tailhook](https://github.com/tailhook/)
+- [DockerBuildManagement](https://github.com/DIPSAS/DockerBuildManagement) - Build Management is a python application, installed with pip. The application makes it easy to manage a build system based on Docker by configuring a single yaml file describing how to build, test, run or publish a containerized solution.
```
Comment (veggiemonk) [inline README.md]: This should be order alphabetically (from A to Z).

Comment (hansehe) [reply README.md]: Thanks, and it should be ok now. Please have a look

File: README.md [hunk 2]
```diff
@@ -22,6 +22,9 @@ All the links are monitored and tested with [awesome_bot](https://github.com/dkh
 
 <!-- TOC -->
 
+- [Awesome Docker ![Awesome](https://github.com/sindresorhus/awesome) ![Join the chat at https://gitter.im/veggiemonk/awesome-docker](https://gitter.im/veggiemonk/awesome-docker) ![Say Thanks](https://saythanks.io/to/veggiemonk)](#awesome-docker-awesomehttpsgithubcomsindresorhusawesome-join-the-chat-at-httpsgitterimveggiemonkawesome-dockerhttpsgitterimveggiemonkawesome-docker-say-thankshttpssaythanksiotoveggiemonk)
```
Comment (veggiemonk) [inline README.md]: ??

Comment (hansehe) [reply README.md]: It should have only been a few line changes, so something have happened. I'm gonna check it out.

Comment (hansehe) [reply README.md]: Reverted the changes and fixed the alphabetical order again. Something happened with the table of content section when hitting save, but discarded these changes.

**Comment by veggiemonk** [comment]
@hansehe Nice! Thank you for your patience.

**Pull request merged**